source/developer/mobile-developer-setup.rst

```diff
@@ -855,3 +855,23 @@ Error message
 
 Solution
   The development version of the Mattermost app cannot be installed alongside a release version. Open ``android/app/build.gradle`` and change the applicationId from ``"com.mattermost.react.native"`` to a unique string for your app.
+
+Errors When Running 'make run-ios'
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+I just got a new machine and downloaded XCode. I used git to clone the React Native Mattermost project from Github. I have everything ready to go. I run ``make run-ios`` and see
```

Comment:
This part isn't needed since the heading covers this

Reply:
sure I can remove this